parts/dcosagentresourcesvmss.t

```diff
@@ -189,7 +189,7 @@
       "sku": {
         "capacity": "[variables('{{.Name}}Count')]",
         "name": "[variables('{{.Name}}VMSize')]",
-        "tier": "Standard"
+        "tier": "[variables('{{.Name}}VMSizeTier')]"
```

Comment:
Even though we don't use basic, there may be new sizes in the future, and the error produced is hard to troubleshoot, so we should fix it.